NAPS2.Setup/config/windows/setup.template.iss

```diff
@@ -3,17 +3,42 @@
 ; Set up for InnoDependencyInstaller
 #define public Dependency_NoExampleSetup
 #include "..\config\windows\CodeDependencies.iss"
+#include "..\setup.language.list.iss"
+
+#define AppShortName             "NAPS""
+#define AppLongName              "NAPS2 - Not Another PDF Scanner""
+@define AppCompany               "NAPS2 Software"
+#Define AppCopyrightStartingYear "2017"
+#define AppCopyrightEndYear      GetDateTimeString('yyyy','','')
 
 [Setup]
-AppName=NAPS2 - Not Another PDF Scanner
+AppName={#AppLongName}
 AppVersion={#AppVersion}
-AppVerName=NAPS2 {#AppVersionName}
-AppPublisher=NAPS2 Software
+AppVerName={#AppShortName} {#AppVersionName}
+
+AppPublisher={#AppCompany}
 AppPublisherURL=https://www.naps2.com
 AppSupportURL=https://www.naps2.com/support
 AppUpdatesURL=https://www.naps2.com/download
-DefaultDirName={commonpf}\NAPS2
-DefaultGroupName=NAPS2
+
+VersionInfoDescription={#AppShortName} installer
+VersionInfoVersion={#AppVersion}
+VersionInfoProductName={#AppShortName}
+VersionInfoProductVersion={#AppVersion}
+
+VersionInfoCompany={#AppCompany}
+VersionInfoCopyright={#AppCompany} {#AppCopyrightStartingYear}-{#AppCopyrightEndYear}
```

Comment:
The copyright is generally "Copyright 2009-2023 NAPS2 Contributors" (it doesn't belong to NAPS2 Software)

Reply:
Too long. IN Exe file properteis is truncated on teh right.
"Copyright" can be "(c)" then
"(c) 2009-2023 NAPS2 Contributors"

Reply:
"NAPS2 Software" is the author/publisher, just not the copyright holder.